packages/plugins/content/slate/src/__tests__/serialize.test.ts

```diff
@@ -243,14 +247,18 @@ describe('serialize to html', () => {
   ].forEach((c, k) => {
     describe(`test case ${k}`, () => {
       it('should serialize properly', () => {
-        // tslint:disable-next-line:no-any
-        expect(html.serialize(Value.fromJSON(c.i as any))).toEqual(c.o);
+        expect(
+          // tslint:disable-next-line:no-any
+          serializationFunctions.slateToHtml(Value.fromJSON(c.i as any))
+        ).toEqual(c.o);
```

Comment:
@PeterKottas you are referring to wrong diffs like this, right?

looks indeed like the previous version was without prettier or with a different configuration.

Or the line length is different